src/tasks/Microsoft.NET.Sdk.WebAssembly.Pack.Tasks/ComputeWasmPublishAssets.cs

```diff
@@ -576,7 +576,7 @@ private void ApplyPublishProperties(ITaskItem newAsemblyAsset)
             }
 
             var extension = candidate.GetMetadata("Extension");
-            if (string.Equals(extension, ".dll", StringComparison.Ordinal) || string.Equals(extension, ".webcil", StringComparison.Ordinal))
+            if (string.Equals(extension, ".dll", StringComparison.Ordinal) || string.Equals(extension, ".webcil", StringComparison.Ordinal) || string.Equals (extension, Utils.WebcilInWasmExtension, StringComparison.Ordinal))
```

Comment:
For blazor, we plan to support both `.wasm` and .`webcil` extensions, but in `WasmAppBuilder` we always use `.wasm`. Is that intended?